### ISSUE TITLE
Standardize category headers

### ISSUE THREAD

**Issue by markmhx** [blockstack/app.co#179]
For some reason, category headers are different between the homepage and their dedicated category pages.

For example, for social networking, it appears as "Hot Social Apps" on the homepage but "Social Networking" on its category page:

<img width="834" alt="screen shot 2018-08-21 at 17 51 33" src="https://user-images.githubusercontent.com/28991/44413154-e6acc980-a56a-11e8-92fb-317022e573b0.png">
<img width="848" alt="screen shot 2018-08-21 at 17 52 00" src="https://user-images.githubusercontent.com/28991/44413161-ead8e700-a56a-11e8-8129-bf44047a85d2.png">

And sometimes it's just a matter of different casing, such as for popular decentralized apps.

<img width="586" alt="screen shot 2018-08-21 at 17 53 59" src="https://user-images.githubusercontent.com/28991/44413251-37bcbd80-a56b-11e8-89f0-948f91c379e1.png">
<img width="565" alt="screen shot 2018-08-21 at 17 54 00" src="https://user-images.githubusercontent.com/28991/44413252-37bcbd80-a56b-11e8-839a-251d746ed170.png">

@jeffdomke @hstove thoughts on how we might want to standardize these?

**Comment by aulneau** [comment]
A lot of this comes from these being partially hard coded in. The featured sections on the home page are manually curated and typed in, whereas the other categories come from the database. I think ideally we'd be pulling all data from the api rather than manually adding/removing items. The more we are manually typing things in, the less likely it is we'll be consistent.

**Comment by markmhx** [comment]
I agree with the suggestion we should keep this data as API-driven as possible.

Perhaps in this case, we'd do best to have an attribute for categories and platforms like "header" that, if available via the API, is used for section headers in lieu of their simple name values.

E.g. if the "Blockstack" platform has name "Blockstack" and header "Popular Blockstack Apps", then the latter would be used. But if not, the former is the default fallback.

This would make sense if headers for categories and platforms are *always* used to indicate the same type of view (such as ordering content by popularity), which I believe is the case and will remain so for the foreseeable future. That way, we use the "header" value when available for the homepage, dedicated category / platform page, and anywhere else we want to show a category or platform's apps.

@hstove what do you think?
